api/app/schemas/forecasts.py

```diff
@@ -10,9 +10,9 @@ class NoonForecastValue(BaseModel):
     """ Data structure for a noon forecast retrieved from BC FireWeather Phase 1 """
     datetime: datetime
     temp_valid: bool
-    temperature: int
+    temperature: int = None
     rh_valid: bool
-    relative_humidity: int
+    relative_humidity: int = None
```

Comment:
We'll have to update the frontend schema for this too yeah?

Reply:
Technically, though, it seems to work. I think it goes into web/src/api/forecastAPI.ts:NoonForecastValue ; but somehow web/src/api/forecastAPI.ts:getNoonForecasts isn't complaining / giving errors.

You want me to go down the rabbit hole - or just leave as is?

Reply:
We're not doing it for ANY of the other values - so I wouldn't want to do it for just those. Right now it's actually consistent with the rest.

Reply:
If we're not doing it for any other optional values then I'd say leave it.

Reply:
Blegh. So there's a bigger fix to be done here. Not going to go down that rabbit hole right now - I don't see the payoff if no-one is using the product. I'm only fixing this so that scary errors don't scare me right now.